<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Arinc665Qt::ExportMediaSetSettingsWidget</class>
 <widget class="QGroupBox" name="Arinc665Qt::ExportMediaSetSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string>Export Media Set Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="arinc665VersionLabel">
     <property name="text">
      <string>ARINC 665 Version</string>
     </property>
     <property name="buddy">
      <cstring>arinc665Version</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="arinc665Version"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="loadHeaderCreationLabel">
     <property name="text">
      <string>Load Header Creation</string>
     </property>
     <property name="buddy">
      <cstring>loadHeaderCreation</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="loadHeaderCreation"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="batchFileCreationLabel">
     <property name="text">
      <string>Batch File Creation</string>
     </property>
     <property name="buddy">
      <cstring>batchFileCreation</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="batchFileCreation"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
